gax-httpjson/src/test/java/com/google/api/gax/httpjson/InstantiatingHttpJsonChannelProviderTest.java

```diff
@@ -94,4 +99,16 @@ public void basicTest() throws IOException {
     // Make sure we can create channels OK.
     provider.getTransportChannel().shutdownNow();
   }
+
+  @Override
+  protected Object getMtlsObjectFromTransportChannel(MtlsProvider provider)
+      throws IOException, GeneralSecurityException {
+    return InstantiatingHttpJsonChannelProvider.newBuilder()
+        .setEndpoint("localhost:8080")
+        .setMtlsProvider(provider)
+        .setHeaderProvider(Mockito.mock(HeaderProvider.class))
+        .setExecutor(Mockito.mock(Executor.class))
+        .build()
```

Comment:
hard to read. consider introducing at least one more local variable.

Reply:
done